contracts/MarginAccount.sol

```diff
@@ -0,0 +1,189 @@
+pragma solidity =0.6.10;
+
+import {SafeMath} from "./packages/oz/SafeMath.sol";
+
+/**
+ * SPDX-License-Identifier: UNLICENSED
+ * @dev The MarginAccount is a library that provides Controller with an Account of Vault structs, and
+ * the functions that manipulate vaults. Vaults describe positions that users have.
+ */
+library MarginAccount {
+    using SafeMath for uint256;
+
+    /**
+     * @dev Account is a struct corresponding to a user that describes how many vaults that user has.
+     */
+    struct Account {
+        //@dev the owner whose vaults we are looking for
```

Comment:
do we want to use @dev or @notice for these? Need to keep everything consistent :) @antoncoding @haythem96 thoughts?

Reply:
I think this is a kind of technical description, so it is okay to use @dev